frb_example/pure_dart/rust/src/lib.rs

```diff
@@ -1,5 +1,9 @@
+/// impl_trait: |Debug|fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result
+/// handle_opaque: HideData
+/// add_box: ExoticOptionals
```

Comment:
Hmm maybe we should not have these (auto-generated) lines

Reply:
Although I'd like to...
But I don't know how to solve it yet. I don't think I'll know for a long time

It's hard for me to know the sig of a Trait,
And what types are private,
And whether the Enum is too large  -:/

Reply:
Then maybe we should move it to `bridge_generated.rs`. In other words, flutter_rust_bridge should not write to anywhere except bridge_generated.rs

Reply:
Oh, do you mean these lines should be written by humans (users), or these are auto generated by flutter_rust_bridge?

If manually written, please consider using the `frb` macro.
If auto generated, see above - we should only write to bridge_generated.rs

Reply:
frb macro maybe I should choose

Reply:
But I need a little more time, or more information